website/common/script/content/spells.js

```diff
@@ -62,7 +62,7 @@ spells.wizard = {
     cast (user, target) {
       each(target, (member) => {
         let bonus = statsComputed(user).int;
-        if (user._id !== member._id) {
+        if (user._id !== member._id && member.stats.class !== 'wizard') {
```

Comment:
I guess we don't want to throw an error, but this way we're just ignoring the mages and on mobile there won't be any notification about this. What do you think @Alys ?

Reply:
There is any way to send notifications to mobile?

Reply:
not really, I mean yes but without proper handling it would be displayed.

@phillipthelen @TheHollidayInn  what do you think?